codespell_lib/data/dictionary.txt

```diff
@@ -205,6 +222,7 @@ achor->anchor
 achored->anchored
 achoring->anchoring
 achors->anchors
+aci->acpi
```

Comment:
not sure about this one, may return false positives

Reply:
Well, anyway I don't see any valid meaning for "aci", in general pourpose.
So, I should let it in, but tell me.

Reply:
Assuming this is https://en.wikipedia.org/wiki/ACPI then can you capitalise the RHS please.